UnityProject/Assets/Scripts/Util/SweetExtensions.cs

```diff
@@ -26,7 +26,7 @@ public static ItemAttributesV2 Item(this GameObject go)
 	public static string ExpensiveName(this GameObject go)
 	{
 		var item = go.Item();
-		if (item != null && !String.IsNullOrWhiteSpace(item.ArticleName)) return item.ArticleName;
+		if (item != null && !String.IsNullOrWhiteSpace(item.ArticleName)) return item.ArticleName.Replace("(Clone)","");
```

Comment:
hmm, I think this is more of a bandaid for a deeper issue. ArticleName isn't supposed to have `(Clone)` in its name in the first place. maybe you could keep it as is for the time being

Reply:
@fomalsd 
> hmm, I think this is more of a bandaid for a deeper issue. ArticleName isn't supposed to have (Clone) in its name in the first place. maybe you could keep it as is for the time being

The code is as follows before the fix:
```csharp
public static string ExpensiveName(this GameObject go)
{
    var item = go.Item();
    if (item != null && !String.IsNullOrWhiteSpace(item.ArticleName)) return item.ArticleName

    var player = go.Player();
    if (player != null && !String.IsNullOrWhiteSpace(player.Name)) return player.Name;

    return go.name.Replace("NPC_", "").Replace("_", " ").Replace("(Clone)","");
}
```

Any GasContainer is considered an item.

The way I'm displaying the name is:
`externalTankName.SetValue = insertedContainer.ExpensiveName();`

which is displaying (Clone) at the end, because anything that returns as an item doesn't filter out (Clone).

I see a few solutions to this:
-Add a replace for (Clone) for items, which was my original solution
-Revert the replacement for (Clone) on items and use `netLabel.SetValue = item.Item().InitialName;`
instead of `netLabel.SetValue = item.ExpensiveName();`

Would the latter be preferred? Seems logical to me, and works in testing, but I'm curious if there's an issue I'm missing with it.

Reply:
Fixed:
Reverted the replacement for (Clone) on items in SweetExtensions.cs and use `netLabel.SetValue = item.Item().InitialName;`
instead of `netLabel.SetValue = item.ExpensiveName();`